README.md

```diff
@@ -24,14 +24,51 @@ Run the test with `ruby path/to/the_test.rb`.
 At the moment the Ruby problems `skip` all but the first test, in order to not
 overwhelm people with errors.
 
-If you want to temporarily disable the skips while working on a test suite, you can
-run the file with a shim that temporarily disables them:
+If you want to temporarily disable the skips while working on a test
+suite, you can run the file with a shim that temporarily disables them:
 
+```sh
+ruby -I./lib -rdisable_skip exercise/exercise/filename_test.rb
+```
+
+It is simpler to use the `make` tool which is available in the project
+root.  It will disable the skip calls for you automatically, it does the
+same thing as the above.
+
+If you would like to use the `make` tool to run a single test, while
+developing clock, for example, you can do something like this:
+
+```sh
+ASSIGNMENT=clock ARGS='-p' make test-assignment
+```
+
+Where, "ASSIGNMENT" is the variable that holds the name of the lesson,
+clock.
+
+ARGS is where you can put additional arguments, here I am demonstrating
+the `-p` argument which may give you 'pride' output.
 
-```ruby
-ruby -I../lib -rdisable_skip <filename_test.rb>
+If you use zsh, you can use the following function to make this process
+simple, shown here using the additional argument `-p` which may run your
+tests with color from the pride reporter with minitest.
```

Comment:
Since you show the command with two different examples of arguments later, I think this can be shortened. Maybe

```
If you use zsh, you can use the following function to make this process simple.
```